controllers/resetPasswordController.js

```diff
@@ -0,0 +1,97 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import models from '../models';
+import sendEmail from '../helpers/sendEmail';
+import Auth from '../helpers/auth';
+
+dotenv.config();
+
+const { users } = models;
+
+const { SECRET } = process.env;
+const expirationTime = {
+  expiresIn: '1day',
+};
+
+/**
+ * @user controller
+ * @exports
+ * @class
+ */
+class ResetPasswordController {
+  /**
+   * Send password reset email.
+   * @param {object} req request
+   * @param {object} res response.
+   * @returns {object} response.
+   */
+  static async sendResetLinkEmail(req, res) {
+    const user = {
+      email: req.body.email,
+    };
+    try {
+      const checkUser = await users.findOne({
+        where: {
+          email: user.email,
+        }
+      });
+      if (checkUser) {
+        const payload = {
+          email: checkUser.email,
+        };
+        const token = jwt.sign(payload, SECRET, expirationTime);
+        req.body.token = token;
+        req.body.template = 'resetPassword';
+        sendEmail(user.email, token, 'resetPassword');
+        return res.status(200).send({ message: 'We have e-mailed a password reset link, Check your email!' });
+      }
+      return res.status(404).json({ error: 'The email provided does not exist' });
+    } catch (error) {
+      return res.status(500).json({ error });
```

Comment:
its not advisable to return an error object in that way, its better to specify the error you want returned